src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
```

Comment:
suggestion

          import { ApplicationConfig } from '@angular/core';

          Explantion: Importing ApplicationConfig seems correct as it is necessary for the configuration of the application. However, it can be beneficial to review this import in conjunction with its usage in the application.